docs/faq.md

```diff
@@ -8,26 +8,17 @@ has_children: false
 
 # FAQ
 
-### 1. What open source license are you using?
-lakeFS is completely free and open source and licensed under the [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0) License.
+### 1. Is lakeFS open source?
+lakeFS is completely free and open source and licensed under the [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0) License. We maintain a public [product roadmap](https://docs.lakefs.io/understand/roadmap.html) and [Slack channel](https://join.slack.com/t/lakefs/shared_invite/zt-ks1fwp0w-bgD9PIekW86WF25nE_8_tw) for open discussions.
```

Comment:
I think you should use https://lakefs.io/slack for the Slack link, which is automatically updated with the invite token.